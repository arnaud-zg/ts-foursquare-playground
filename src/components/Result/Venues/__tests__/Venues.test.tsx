import React from 'react'
import { Provider } from 'react-redux'
import renderer from 'react-test-renderer'
import { configureStore } from 'ts-foursquare'
import { Venues } from '..'

const store = configureStore()

describe('Component/Venues', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Venues />
        </Provider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
