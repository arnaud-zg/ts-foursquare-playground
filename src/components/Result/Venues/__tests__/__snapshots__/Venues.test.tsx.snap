// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/Venues renders correctly 1`] = `
<div>
  <div>
    <h3>
      Venues
    </h3>
    <code>
      <pre>
        {}
      </pre>
    </code>
  </div>
</div>
`;
